zip_loader.py

```diff
@@ -516,36 +514,36 @@ def upload_zip(source_name, output_directory):
 
     # Upload to drive
     load_zip_to_drive(feature, 'gdb_id', new_gdb_zip, feature['parent_ids'])
-    print 'GDB loaded'
+    print('GDB loaded')
     load_zip_to_drive(feature, 'shape_id', new_shape_zip, feature['parent_ids'])
-    print 'Shape loaded'
+    print('Shape loaded')
     load_zip_to_drive(feature, 'hash_id', new_hash_zip, [HASH_DRIVE_FOLDER])
-    print 'Hash loaded'
+    print('Hash loaded')
 
     spec_manager.save_spec_json(feature)
 
 
 def delete_feature(source_name):
     """Delete a feature, remove it from packages and delete all it's files on drive."""
-    print 'Deleting', source_name
-    confirm_delete = raw_input('Are you sure you want to permanently delete files (yes, no): ')
+    print('Deleting', source_name)
+    confirm_delete = input('Are you sure you want to permanently delete files (yes, no): ')
```

Comment:
this makes it so that python 2 will never work. is that ok?

Reply:
Yes. I don't see a reason to go back to python 2.